src/containers/ProfilePage/index.jsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { Container, Row } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import UserDetail from "../../components/profile/UserDetail";
+import UserEmail from "../../components/profile/UserEmail";
+import UserPassword from "../../components/profile/UserPassword";
+import defaultProfileImage from "../../images/defaultProfileImage.png";
+import "./index.scss";
+
+function getArrowClass(selectedElement, currentElement) {
+  return selectedElement === currentElement ? "arrow down" : "arrow right";
+}
+
+function checkCollapseClass(selectedElement, currentElement) {
```

Comment:
for check naming, I was thinking that this function will check if the element should collapsed or not and then add the appropriate class. I will changed it.

for returning `null` or `undefined`, it will add to the class name like:
![image](https://user-images.githubusercontent.com/30551053/99356628-8462e080-28bb-11eb-952a-fab9195d4b65.png)

so I returned an empty string. I think I might return another unused class like "notCollapsed" instead of empty string, is that a good insulation? @louisrli

Reply:
getCollapseClass() || ""

in isolation the function should return null to represent empty. this is the "clean" representation of something not being present. then use the null result as needed

Reply:
I see. btw, I fixed that after refactoring the code. I think the code now is clean.